docs/core/deploying/index.md

```diff
@@ -1,95 +1,104 @@
 
-It also has a number of disadvantages:
+The executable binary produced for the specified target platform. For example, if you have an app named **word_reader**, and you publish a self-contained executable for Windows, a *word_reader.exe* file is created. Publishing for Linux or macOS, a *word_reader* file is created. The target platform and architecture is specified with the `-r <RID>` parameter. For more information about RIDs, see [.NET Core RID Catalog](../rid-catalog.md).
 
-- Because .NET Core is included in your deployment package, you must select the target platforms for which you build deployment packages in advance.
+If the app has platform-specific dependencies, such as a NuGet package containing platform-specific dependencies, these are copied to the publish folder along with the app.
 
-- The size of your deployment package is relatively large, since you have to include .NET Core as well as your app and its third-party dependencies.
+A self-contained executable can be created with the [`dotnet publish -r`](../tools/dotnet-publish.md) command.
 
-  Starting with .NET Core 2.0, you can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/runtime/blob/master/docs/design/features/globalization-invariant-mode.md). Ordinarily, .NET Core on Linux relies on the [ICU libraries](http://icu-project.org) for globalization support. In invariant mode, the libraries are not included with your deployment, and all cultures behave like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
+```dotnet
+dotnet publish -r <RID>
+```
 
-- Deploying numerous self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
+### Advantages
 
-## Framework-dependent executables (FDE)
+- **Control .NET Core version**\
+You control which version of .NET Core is deployed with your app.
 
-Starting with .NET Core 2.2, you can deploy your app as an FDE, along with any required third-party dependencies. Your app will use the version of .NET Core that's installed on the target system.
+- **Platform-specific targeting**\
+Because you have to publish your app for each platform, you know where your app will run. If .NET Core introduces a new platform, users can't run your app on that platform until you release a version targeting that platform. You can test your app for compatibility problems before your users run your app on the new platform.
 
-### Why deploy a framework-dependent executable?
+### Disadvantages
 
-Deploying an FDE has a number of advantages:
+- **Larger deployments**\
+Because your app includes the .NET Core runtime and all of your app dependencies, the hard drive space required is greater than a [runtime-dependent](#runtime-dependent) version.
```

Comment:
```suggestion
Because your app includes the .NET Core runtime and all of your app dependencies, the download size and hard drive space required is greater than a [runtime-dependent](#runtime-dependent) version.
```